roblox/bases/basecatalogitem.py

```diff
@@ -0,0 +1,47 @@
+"""
+
+This file contains the BaseCatalogItem object, which represents a Roblox catalog item ID.
+
+"""
+
+from __future__ import annotations
+from typing import TYPE_CHECKING
+
+from .baseitem import BaseItem
+
+if TYPE_CHECKING:
+    from ..client import Client
+
+
+class BaseCatalogItem(BaseItem):
+    """
+    Represents a Roblox instance ID.
+    Instance IDs represent the ownership of a single Roblox item.
+
+    Attributes:
+        id: The item ID.
+        item_type: The item's type, either 1 or 2.
+    """
+
+    def __init__(self, client: Client, catalog_item_id: int):
+        """
+        Arguments:
+            client: The Client this object belongs to.
+            catalog_item_id: The ID of the catalog item.
+        """
+
+        self._client: Client = client
+        self.id: int = catalog_item_id
+        self.item_type: int = catalog_item_type
```

Comment:
Where is this coming from?

Reply:
I was in the process of converting the `itemType` stuff to a class, I committed this on accident before the rest was ready